web/src/config.ts

```diff
@@ -1,4 +1,6 @@
 export const config = {
   pollingRate: 5000, // Frequency with which updates are requested from the server
-  csrfPollingRate: 60000 // Time the sever waits before polling new csrf token.
+  csrfPollingRate: 60000, // Time the sever waits before polling new csrf token.
+  serverURLProd: "https://batailley.informatik.uni-kiel.de:50003/api", // PROD URL to direct player qr-codes
```

Comment:
currently the web URL which is wrong. which port do we use for the game backend? 5002 as well?